MMR.Randomizer/Templates/HtmlSpoiler.cs

```diff
@@ -26,60 +26,41 @@ public partial class HtmlSpoiler : HtmlSpoilerBase
         /// </summary>
         public virtual string TransformText()
         {
-            this.Write(@"<html>
-<head>
-<style>
-    body.dark-mode {
-      background-color: #111;
-      color: #ccc;
-    }
-    body.dark-mode a {
-      color: #111;
-    }
-    body.dark-mode button {
-      background-color: #ddd;
-      color: #111;
-    }
-
-    body.light-mode {
-      background-color: #eee;
-      color: #111;
-    }
-    body.light-mode a {
-      color: #111;
-    }
-    body.light-mode button {
-      background-color: #111;
-      color: #ccc;
-    }
-
-    th{ text-align:left }
-    .region { text-align: center; font-weight: bold; }
-    [data-content]:before { content: attr(data-content); }
-
-    .dark-mode .spoiler{ background-color:#ccc }
-    .dark-mode .spoiler:active { background-color: #111;  }
-    .dark-mode .show-highlight .unavailable .newlocation { background-color: #500705; }
-    .dark-mode .show-highlight .acquired .newlocation { background-color: #69591f; }
-    .dark-mode .show-highlight .available .newlocation { background-color: #313776; }
-
-    .light-mode .spoiler{ background-color:#111 }
-    .light-mode .spoiler:active { background-color: #ccc;  }
-    .light-mode .show-highlight .unavailable .newlocation { background-color: #FF9999; }
-    .light-mode .show-highlight .acquired .newlocation { background-color: #99FF99; }
-    .light-mode .show-highlight .available .newlocation { background-color: #9999FF; }
-
-
-    #spoilerLogState { width: 560px; }
-</style>
-</head>
-<body class=""light-mode"">
-<label><b>Version: </b></label><span>");
+            this.Write("<html>\r\n<head>\r\n<style>\r\n    body.dark-mode {\r\n      background-color: #111;\r\n   " +
```

Comment:
More specific about the formatting looking auto-generated, I meant this section here which was changed to look like a later section